var/www/html/tools/MemcachedAdmin/Library/Analysis.php

```diff
@@ -0,0 +1,332 @@
+<?php
+/**
+ * Copyright 2010 Cyrille Mahieux
```

Comment:
I removed them. We could possibly try to add them back, but we would would have to think about how we make that happen. it would be strange with the nginx load balncer you wouldn't know which container you were hitting and thus debugging with these tools, and the backend container is likely to change between page loads

It would probably be best to bake this into the image if we want to readd them, but that might make securing them with a unique password a challenge...